diff --git a/node_modules/react-native/scripts/react_native_pods.rb b/node_modules/react-native/scripts/react_native_pods.rb
index 5063116..4f90ccb 100644
--- a/node_modules/react-native/scripts/react_native_pods.rb
+++ b/node_modules/react-native/scripts/react_native_pods.rb
@@ -79,7 +79,8 @@ def use_react_native! (
   flipper_configuration: FlipperConfiguration.disabled,
   app_path: '..',
   config_file_dir: '',
-  ios_folder: 'ios'
+  ios_folder: 'ios',
+  privacy_file_aggregation_enabled: false # accept for Expo/newer RN compat, unused in 0.73
 )
 
   # Set the app_path as env variable so the podspecs can access it.
